### PR TITLE
docs: Update "Users" part of README.md

### PR DESCRIPTION
**Pull request by nitkon**
Fixes: #36

Signed-off-by: Nitesh Konkar <niteshkonkar@in.ibm.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,22 +27,8 @@ Kata Containers is working to build a global, diverse and collaborative communit
...
-that interest you.
-
-### Installing Clear Containers
-
-See https://github.com/clearcontainers/runtime/wiki/Installation.
-
-### Installing runV
-
-See https://github.com/hyperhq/runv.
+See [Kata Containers installation user guides](https://github.com/kata-containers/documentation/blob/master/install/README.md#kata-containers-installation-user-guides) for details on how to install Kata Containers for your preferred
```
Comment (jodh-intel) [inline README.md]: Hi @nitkon - thanks for raising.

Please can you update the table of contents links at the top of the doc as they still reference the old section names.

@klynnrif - ptal.

Comment (nitkon) [reply README.md]: @jodh-intel @klynnrif Done!

Comment (jodh-intel) [reply README.md]: Thanks - I'd be tempted to remove the `#kata-containers-installation-user-guides` as that's a link to the top of the document anyway.

Comment (nitkon) [reply README.md]: @jodh-intel : Fixed.  Thanks!

**Comment by jodh-intel** [comment]
lgtm

**Comment by jodh-intel** [comment]
@klynnrif - ptal.

**Comment by jcvenegas** [comment]
lgtm

**Comment by jodh-intel** [comment]
We just need 1 more ack on this tiny change - @grahamwhaley - could you tal?

**Pull request merged**